cpp/src/arrow/ipc/CMakeLists.txt

```diff
@@ -19,16 +19,50 @@
 # arrow_ipc
 #######################################
 
-# Headers: top level
-install(FILES
-  adapter.h
-  metadata.h
-  memory.h
-  DESTINATION include/arrow/ipc)
+set(ARROW_IPC_LINK_LIBS
+  arrow_io
+  arrow_shared
+)
+
+set(ARROW_IPC_PRIVATE_LINK_LIBS
+  )
+
+set(ARROW_IPC_TEST_LINK_LIBS
+  arrow_ipc
```

Comment:
Roger. This isn't causing immediate problems -- let's create JIRAs to build static libs for the leaf libraries too